docs(root): copied prop descriptions/explainations to code examples (#1602)
mi6/ic-design-system: src/content/structured/components/date-input/code.mdx

### Min and max

> It is possible to set a min and max date range. If a date is set and it is before the min or after the max date, a validation error will appear.

Reviewer: ```suggestion
It is possible to set a minimum and maximum date range. If a date is set and it is before the minimum or after the maximum date, a validation error will appear.
```

> The min or max dates accepts the same date formats as the data-input component.

Reviewer: I feel like this sentence doesn't completely make sense anyway? Should it be something like this:

```suggestion
The `min` and `max` props accept the same date formats as the `value` prop.
```
Unless I've misunderstood
